ansible-ipi-install/roles/installer/defaults/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
 # defaults file for installer
+use_rhcos_image_cache: true
```

Comment:
its complaining about no newline at end of file here.

Reply:
Also, I'd think we wouldn't want this set to true by default?

Reply:
Ack